wazuh/wazuh
# Nodejs translation efficacy test

|Related issue|
|---|
|https://github.com/wazuh/intelligence-data/issues/275|



## Description

This PR adds a new efficacy test to verify that the nodejs translation for Windows platform works corectly

## QA tests

Due to the misalignment between 4.8.0 and 4.9.0 we run this tests locally in 4.8.0 to see they pass
![image](https://github.com/wazuh/wazuh/assets/13010397/84af3253-3d20-44f0-8b3a-0ae09241e629)
